CPPLINT.cfg

```diff
@@ -0,0 +1,13 @@
+# SPDX-FileCopyrightText: 2023 Carl Zeiss Microscopy GmbH
+#
+# SPDX-License-Identifier: LGPL-3.0-or-later
+set noparent
+linelength=200
+# Filter Rationale
+# -whitespace: Largely checked by editorconfig
+# +whitespace/tab: Back up to editorconfig since some were not found
+# -legal/copyright: Taken care of by REUSE
+# -runtime/*: Fix in separate PR or leave as-is
+# -build/*: Fix in separate PR or leave as-is
+# -readability/*: Fix in separate PR or leave as-is
+filter=-whitespace,+whitespace/tab,-runtime/explicit,-runtime/references,-runtime/int,-runtime/printf,-runtime/threadsafe_fn,-runtime/arrays,-runtime/indentation_namespace,-build/include_what_you_use,-build/include,-build/header_guard,-build/c++11,-build/include_subdir,-build/namespaces,-build/include_order,-build/storage_class,-readability/todo,-readability/inheritance,-readability/casting,-readability/utf8,-readability/braces,-readability/namespace,-readability/multiline_comment,-legal/copyright
```

Comment:
@ptahmose I just saw that you already fixed some of the cpplint issues under _Src/libCZI_ represented by filters in this file. I still like to keep them to ultimately apply some of those fixes to other subfolders of _Src_ as well (first and foremost _CZICmd_) - however tracked in #35 .

Reply:
LGTM, thanks!